src/components/ReportCard/ReportCard.js

```diff
@@ -2,17 +2,38 @@ import React from 'react';
 import './ReportCard.scss';
 import PropTypes from 'prop-types'
 
-const ReportCard = ({ districtName, studentTeacherRatio, instructionSalaryPercentOfTotal, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
+const ReportCard = ({ districtName, studentTeacherRatio, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
```

Comment:
why did we get rid of this instructional salary percent of total. We agreed to display this yesterday.